### PR TITLE
add(tem): dmarc documentation pages

### PR DESCRIPTION
**Pull request by nerda-codes**
- Added concepts 
- How to add DMARC doc
- DMARC configuration reference content

### CODE REVIEW

File: managed-services/transactional-email/how-to/add-dmarc-record.mdx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com/).
+  - You have a domain name.
+  - You have [configured your domain with Transactional Email](/managed-services/transactional-email/how-to/configure-domain-with-transactional-email).
+  - You have configured SPF and DKIM records on your domain.
+</Message>
+
+1. Click **Transactional Email** in the **Managed Services** section of the [console](https://console.scaleway.com/) side menu.
+2. Click your domain. You are redirected to the records configuration page.
```
Comment (bene2k1) [inline managed-services/transactional-email/how-to/add-dmarc-record.mdx:29]:
```suggestion
2. Click your domain name. You are redirected to the records configuration page.
```

**Review by bene2k1 — APPROVED** [review]

File: managed-services/transactional-email/reference-content/dmarc-configuration.mdx [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+| `ri`    | Indicates a request to receivers to generate aggregate reports separated by the requested number of seconds. | No        | Number in seconds.                                                                                                                                                 | `86400`         |
+| `rua`   | Comma-separated addresses to which aggregate feedback is sent.                                               | No        | x                                                                                                                                                                  | x               |
+| `ruf`   | Comma-separated addresses to which message-specific failure information is to be reported.                   | No        | x                                                                                                                                                                  | x               |
+
+## Examples of DMARC configuration
+
+Below are a few examples of DMARC configuration scenarios you can get inspired from to [configure your DMARC](/managed-services/transactional-email/how-to/add-dmarc-record/).
```
Comment (RoRoJ) [inline managed-services/transactional-email/reference-content/dmarc-configuration.mdx:63]:
```suggestion
Below are a few examples of DMARC configuration scenarios for inspiration to [configure your DMARC](/managed-services/transactional-email/how-to/add-dmarc-record/).
```

**Review by RoRoJ — APPROVED** [review]

File: managed-services/transactional-email/how-to/add-dmarc-record.mdx [hunk 3]
```diff
@@ -0,0 +1,46 @@
+---
+meta:
+  title: How to configure a DMARC record
+  description: This page explains how to configure a DMARC record
+content:
+  h1: How to configure a DMARC record
+  paragraph: This page explains how to configure a DMARC record
+tags: spf spf-record dkim dkim-records transactional
```
Comment (jerjako) [inline managed-services/transactional-email/how-to/add-dmarc-record.mdx:8]:
```suggestion
tags: spf spf-record dkim dkim-records dmarc dmarc-record transactional
```
Are we using "-record" or "-records"?

File: managed-services/transactional-email/how-to/add-dmarc-record.mdx [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+This page shows you how to configure a [DMARC record](/managed-services/transactional-email/concepts/#dmarc-record) on your domain from the **Scaleway console**.
+If you have registered a domain at another provider, you can still use this documentation, as the process is similar.
+
+<Macro id="iam-requirements" />
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com/).
+  - You have a domain name.
+  - You have [configured your domain with Transactional Email](/managed-services/transactional-email/how-to/configure-domain-with-transactional-email).
+  - You have configured SPF and DKIM records on your domain.
```
Comment (jerjako) [inline managed-services/transactional-email/how-to/add-dmarc-record.mdx:25]: The DMARC record will be added as long as SPF and DKIM records. I think we can remove this line.

File: managed-services/transactional-email/how-to/add-dmarc-record.mdx [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+</Message>
+
+1. Click **Transactional Email** in the **Managed Services** section of the [console](https://console.scaleway.com/) side menu.
+2. Click your domain. You are redirected to the records configuration page.
+3. Add a `TXT` record type in the root zone of your domain on your DNS zone.
```
Comment (jerjako) [inline managed-services/transactional-email/how-to/add-dmarc-record.mdx:30]:
```suggestion
3. Add a `TXT` record type in the same zone of your TEM domain on your DNS zone.
```

The domain registered at TEM is not necessarily in the root zone. 

For example, I have "mydomain.tld". And a zone "tem", I can register a TEM domain "tem.mydomain.tld" or "stg.tem.mydomain.tld".
In the first case, I'm in the root record of the "tem" zone. In the second case, I'm not a root record on the "tem" zone.
Note that this line is also in another tem how-to

File: managed-services/transactional-email/reference-content/dmarc-configuration.mdx [hunk 6]
```diff
@@ -0,0 +1,135 @@
...
+
+This documentation provides information about the mandatory and optional tags to configure when [adding a DMARC record](/managed-services/transactional-email/how-to/add-dmarc-record/) to your domain, what they mean, the expected value for them, and their default value. You will also find examples of DMARC configurations according to your use-case.
+
+## DMARC Policy
+
+DMARC policies specify how receiving email servers should handle messages that fail authentication checks (SPF and/or DKIM) for the domain.
+The three main DMARC policies are:
+
+- none: No action is taken by the receiving mail servers based on DMARC authentication results. This policy allows domain owners to assess the impact of DMARC enforcement on their email traffic without risking legitimate emails being blocked or sent to spam. It is used for monitoring and collecting data without impacting email delivery.
```
Comment (jerjako) [inline managed-services/transactional-email/reference-content/dmarc-configuration.mdx:27]:
```suggestion
- none: no action is taken by the receiving mail servers based on DMARC authentication results. This policy allows domain owners to assess the impact of DMARC enforcement on their email traffic without risking legitimate emails being blocked or sent to spam. It is used for monitoring and collecting data without impacting email delivery.
```
Just to be consistent with the 2 other policy

File: managed-services/transactional-email/reference-content/dmarc-configuration.mdx [hunk 7]
```diff
@@ -0,0 +1,135 @@
...
+|---------|-------------------------------------------------------------------------------------------|-----------|--------------------------------------------|---------------|
+| `v`     | Indicates the DMARC version.                                                              | Yes       | `DMARC1`                                   | x             |
+| `p`     | Indicates the policy to apply when/if a check fails.                                      | Yes       | `none`, `quarantine`, or `reject`          | x             |
+| `sp`    | Indicates the subdomain policy to apply when/if a check fails.                            | No        | `none`, `quarantine`, or `reject`          | x             |
+| `adkim` | Defines the DKIM alignment mode.                                                          | No        | `r (relaxed mode)` or `s (strict mode)`    | `r`           |
+| `aspf`  | Defines the SPF alignment mode.                                                           | No        | `r (relaxed mode)` or `s (strict mode)`    | `r`           |
```
Comment (jerjako) [inline managed-services/transactional-email/reference-content/dmarc-configuration.mdx:40]:
```suggestion
| `adkim` | Defines the DKIM alignment mode.                                                          | No        | `r` (relaxed mode) or `s` (strict mode)    | `r`           |
| `aspf`  | Defines the SPF alignment mode.                                                           | No        | `r` (relaxed mode) or `s` (strict mode)    | `r`           |
```

IMO we should only backtick the values that can be in the tag

File: managed-services/transactional-email/reference-content/dmarc-configuration.mdx [hunk 8]
```diff
@@ -0,0 +1,135 @@
...
+
+## DMARC Reporting
+
+DMARC reporting involves the collection and analysis of DMARC reports generated by receiving mail servers.
+These reports provide domain owners with valuable insights into their email authentication status, including authentication successes, failures, and sources of spoofed emails.
+There are two main types of DMARC reports:
+
+- aggregate reports (`rua`): aggregate reports provide data about a domain's email authentication activity. These reports include information such as the volume of emails received, the percentage of emails that pass or fail DMARC checks, and the sources of failed authentication.
+
+- forensic reports (`ruf`): forensic reports provide detailed information about individual email messages that fail DMARC authentication. They include the complete headers and body of the failed emails, details of the authentication checks performed, and any errors encountered.
```
Comment (jerjako) [inline managed-services/transactional-email/reference-content/dmarc-configuration.mdx:51]:
```suggestion
- aggregate reports (`rua`): aggregate reports provide data about a domain's email authentication activity. These reports include information such as the volume of emails received, the percentage of emails that pass or fail DMARC checks, and the sources of failed authentication.
- forensic reports (`ruf`): forensic reports provide detailed information about individual email messages that fail DMARC authentication. They include the complete headers and body of the failed emails, details of the authentication checks performed, and any errors encountered.
```

Not sure about this one, but I don't see any blank line on other item list

File: managed-services/transactional-email/reference-content/dmarc-configuration.mdx [hunk 9]
```diff
@@ -0,0 +1,135 @@
...
+
+The following DMARC configuration is the most basic one. It is commonly used for monitoring purposes rather than enforcing specific actions on incoming emails.
+
+    ```
+    v=DMARC1; p=none; rua=mailto:mydomain@example.com
+    ```
+
+- `v=DMARC1` indicates the version of the DMARC protocol being used (version 1).
+- `p=none` sets the DMARC policy to "none" meaning that no specific action should be taken based on the DMARC policy. This policy is typically used for monitoring purposes.
+- `rua=mailto:mydomain@example.com` specifies the email address to which aggregate reports should be sent.
```
Comment (jerjako) [inline managed-services/transactional-email/reference-content/dmarc-configuration.mdx:82]:
```suggestion
- `rua=mailto:mydomain@example.com` specifies the email address to which aggregate reports should be sent (`mydomain@example.com`).
```

Add a little reminder on the final value

File: managed-services/transactional-email/reference-content/dmarc-configuration.mdx [hunk 10]
```diff
@@ -0,0 +1,135 @@
...
+
+The following DMARC configuration provides a moderate level of protection against fraudulent emails while still allowing legitimate emails to be delivered.
+
+    ```
+    v=DMARC1; p=quarantine; rua=mailto:mydomain@example.com
+    ```
+
+- `v=DMARC1` indicates the version of the DMARC protocol being used (version 1).
+- `p=quarantine` sets the DMARC policy to "quarantine", meaning that suspicious emails should be treated with caution and possibly moved to the recipient's spam or junk folder.
+- `rua=mailto:mydomain@example.com` specifies the email address to which aggregate reports should be sent.
```
Comment (jerjako) [inline managed-services/transactional-email/reference-content/dmarc-configuration.mdx:97]:
```suggestion
- `rua=mailto:mydomain@example.com` specifies the email address to which aggregate reports should be sent (`mydomain@example.com`).
```
Like above

File: managed-services/transactional-email/reference-content/dmarc-configuration.mdx [hunk 11]
```diff
@@ -0,0 +1,135 @@
...
+### Configuration for strong email security
+
+The following DMARC configuration allows you to prioritize email security, and take strong action against fraudulent or unauthorized emails.
+
+```
+v=DMARC1; p=reject; rua=mailto:mydomain@example.com; ruf=mailto:mydomain-forensic@example.com
+```
+
+- `v=DMARC1` indicates the version of the DMARC protocol being used (version 1).
+-  `p=reject` sets the DMARC policy to "reject", meaning that suspicious emails should be rejected outright and not delivered to the recipient's inbox.
```
Comment (jerjako) [inline managed-services/transactional-email/reference-content/dmarc-configuration.mdx:111]:
```suggestion
- `p=reject` sets the DMARC policy to "reject", meaning that suspicious emails should be rejected outright and not delivered to the recipient's inbox.
```

File: managed-services/transactional-email/reference-content/dmarc-configuration.mdx [hunk 12]
```diff
@@ -0,0 +1,135 @@
...
+- `rua=mailto:mydomain@example.com` specifies the email address to which aggregate reports should be sent.
```
Comment (jerjako) [inline managed-services/transactional-email/reference-content/dmarc-configuration.mdx:112]:
```suggestion
- `rua=mailto:mydomain@example.com` specifies the email address to which aggregate reports should be sent (`mydomain@example.com`).
```

File: managed-services/transactional-email/reference-content/dmarc-configuration.mdx [hunk 13]
```diff
@@ -0,0 +1,135 @@
...
+- `ruf=mailto:mydomain-forensic@example.com` specifies the email address to which forensic reports should be sent.
```
Comment (jerjako) [inline managed-services/transactional-email/reference-content/dmarc-configuration.mdx:113]:
```suggestion
- `ruf=mailto:mydomain-forensic@example.com` specifies the email address to which forensic reports should be sent (`mydomain-forensic@example.com`).
```

File: managed-services/transactional-email/reference-content/dmarc-configuration.mdx [hunk 14]
```diff
@@ -0,0 +1,135 @@
...
+### Strict configuration
+
+The following DMARC configuration shows you how to enforce strict email authentication policies with robust reporting mechanisms.
+
+    ```
+    v=DMARC1; p=reject; rua=mailto:mydomain@example.com; sp=reject; pct=80; fo=1; adkim=s; aspf=s
+    ```
+
+- `v=DMARC1` indicates the version of the DMARC protocol being used (version 1).
+-  `p=reject` sets the DMARC policy to "reject", meaning that suspicious emails should be rejected outright and not delivered to the recipient's inbox.
```
Comment (jerjako) [inline managed-services/transactional-email/reference-content/dmarc-configuration.mdx:126]:
```suggestion
- `p=reject` sets the DMARC policy to "reject", meaning that suspicious emails should be rejected outright and not delivered to the recipient's inbox.
```

File: managed-services/transactional-email/reference-content/dmarc-configuration.mdx [hunk 15]
```diff
@@ -0,0 +1,135 @@
...
+- `rua=mailto:mydomain@example.com` specifies the email address to which aggregate reports should be sent.
```
Comment (jerjako) [inline managed-services/transactional-email/reference-content/dmarc-configuration.mdx:127]:
```suggestion
- `rua=mailto:mydomain@example.com` specifies the email address to which aggregate reports should be sent (`mydomain@example.com`).
```

File: managed-services/transactional-email/how-to/add-dmarc-record.mdx [hunk 16]
```diff
@@ -0,0 +1,46 @@
...
+    Your **Value** field should look similar to the following:
+    ```
+    v=DMARC1; p=none; rua=mailto:your@email.com
+    ```
+6. Click **Add record**.
+
+<Message type="tip">
+  For more in-depth information about DMARC configuration, refer to the [dedicated documentation](/managed-services/transactional-email/reference-content/dmarc-configuration/) page.
+</Message>
+
```
Comment (jerjako) [inline managed-services/transactional-email/how-to/add-dmarc-record.mdx:42]: A note could be added: You can have only one _dmarc TXT records per `Name`
Example: `_dmarc.mydomain.tld` and `_dmarc.tem.mydomain.tld` is valid. but not 2 `_dmarc.mydomain.tld`

File: managed-services/transactional-email/reference-content/dmarc-configuration.mdx [hunk 17]
```diff
@@ -0,0 +1,135 @@
...
+- `pct=80` specifies that the DMARC policy should be applied to 80% of incoming emails.
+- `fo=1` sets the failure options to "1" meaning that suspicious emails should be treated with caution.
+- `adkim=s` sets the DKIM alignment mode to strict. This means that you are requiring that the DKIM authentication mechanism passes with exact alignment, to pass DMARC authentication. This provides a higher level of assurance that the email originated from the claimed sender and has not been tampered with in transit.
+- `aspf=s` sets the SPF alignment mode to strict. This means that you are requiring that the SPF authentication mechanism passes with exact alignment, to pass DMARC authentication. This provides a higher level of assurance that the email originated from the claimed sender and has not been tampered with in transit.
+
+This configuration might be useful if you want to enforce strict authentication policies while also maintaining visibility into email traffic through comprehensive reporting.
+It ensures that only legitimate emails from authenticated sources are delivered while providing insights into potential authentication failures and security threats.
```
Comment (jerjako) [inline managed-services/transactional-email/reference-content/dmarc-configuration.mdx:135]: I think you can add (or edit an existing scenario) with multiple rua or ruf. Like this:

`rua=mailto:mydomain@example.com,mailto:mydomain-dev@example.com`

File: managed-services/transactional-email/how-to/add-dmarc-record.mdx (hunk 3 above)
Comment (nerda-codes) [reply managed-services/transactional-email/how-to/add-dmarc-record.mdx:8]: we can use everything!

File: managed-services/transactional-email/how-to/add-dmarc-record.mdx [hunk 18]
```diff
@@ -0,0 +1,51 @@
...
+5. Enter your desired tags and their values in the **Value** section. Make sure that you add the mandatory `v` and `p` tags, and that you separate tags from each other with semi-colons. Find out about DMARC configuration in our [dedicated documentation](/managed-services/transactional-email/reference-content/dmarc-configuration/).
+    Your **Value** field should look similar to the following
+    ```
+    v=DMARC1; p=none; rua=mailto:your@email.com
+    ```
+    <Message type="important">
+     It is possible to configure several DMARC records for one domain, but they must have a different name to be valid.
+     For example, the `_dmarc.mydomain.com` and `_dmarc.tem.mydomain.com` DMARC records are valid.
+    </Message>
+6. Click **Add record**.
```
Comment (jnolleau) [inline managed-services/transactional-email/how-to/add-dmarc-record.mdx:42]: I don't do suggestions as I don't know all the documentation writing guidelines and good syntax :)
We cannot manage our domain by the TEM page, I would rather say:
1. Click **Domains and DNS** in the **Network** section of the [console](https://console.scaleway.com/) side menu.
2. Click your domain then **DNS Zones** then click the zone where you want to add your DMARC record .

Maybe we can add a Note to say that it is only valid for domains managed by Scaleway. Steps could be different for other providers.

File: managed-services/transactional-email/how-to/add-dmarc-record.mdx [hunk 19]
```diff
@@ -0,0 +1,51 @@
...
+
+<Message type="important">
+  Emails sent from sources not configured in your SPF record, or that do not have a DKIM **will be rejected if you use the `p=reject` policy**.
+</Message>
```
Comment (jnolleau) [inline managed-services/transactional-email/how-to/add-dmarc-record.mdx:46]: I don't actually get why we add this note here, as we don't talk about "p=reject" before.
I would rather see it on the "Understanding DMARC configuration" (where is already is :) )
Or let it here but make it in accordance with the example record " v=DMARC1; p=none; rua=mailto:your@email.com" mentioned above.

File: managed-services/transactional-email/reference-content/dmarc-configuration.mdx [hunk 20]
```diff
@@ -0,0 +1,151 @@
...
+|---------|-------------------------------------------------------------------------------------------|-----------|--------------------------------------------|---------------|
+| `v`     | Indicates the DMARC version.                                                              | Yes       | `DMARC1`                                   | x             |
+| `p`     | Indicates the policy to apply when/if a check fails.                                      | Yes       | `none`, `quarantine`, or `reject`          | x             |
+| `sp`    | Indicates the subdomain policy to apply when/if a check fails.                            | No        | `none`, `quarantine`, or `reject`          | x             |
+| `adkim` | Defines the DKIM alignment mode.                                                          | No        | `r` (relaxed mode) or `s` (strict mode)    | `r`           |
+| `aspf`  | Defines the SPF alignment mode.                                                           | No        | `r` (relaxed mode) or `s` (strict mode)    | `r`           |
```
Comment (jnolleau) [inline managed-services/transactional-email/reference-content/dmarc-configuration.mdx:44]: Maybe we can add a small paragraph or note explaining what is "strict" and what is "relaxed" ?

File: managed-services/transactional-email/reference-content/dmarc-configuration.mdx [hunk 21]
```diff
@@ -0,0 +1,151 @@
...
+| `ruf`   | Comma-separated addresses to which message-specific failure information is to be reported.                   | No        | x                                                                                                                                                                  | x               |
+
+## Examples of DMARC configuration
+
+Below are a few examples of DMARC configuration scenarios for inspiration to [configure your DMARC](/managed-services/transactional-email/how-to/add-dmarc-record/).
+
+<Message type="important">
+
+  - Using the following examples could affect your email deliverability, make sure that you ask for help if you are not familiar with DMARC configuration.
+  - If you need help with DMARC configuration, you can contact the Transactional Email team in the #transactional-email channel on the [Scaleway Community Slack](https://scaleway-community.slack.com).
```
Comment (jnolleau) [inline managed-services/transactional-email/reference-content/dmarc-configuration.mdx:71]: To Define with Mouad and Loïc ?

**Review by jnolleau — COMMENTED** [review]
Amazing doc :100:

**Pull request merged**